src/cmd/run.js

```diff
@@ -44,16 +47,18 @@ export type WatcherCreatorParams = {|
   desktopNotifications?: typeof defaultDesktopNotifications,
```

Comment:
This is not anymore a parameter of the `defaultWatchCreator` utility method, we should remove it from this type as well.